src/controller/SnoozeForm.tsx

```diff
@@ -81,6 +92,10 @@ class SnoozeForm extends Component<Props, State> {
     this.setState({ snoozeReason, duration });
   }
 
+  subReasonChange(subreason: Subreason | "") {
```

Comment:
Noticing a lot of `Subreason | ""` types. Should `""` just be a valid `Subreason` type?

Reply:
I didn't want to add `""` as part of `Subreason` since it is the unset state not a valid subreason so I changed it to use `undefined` instead